deploy/charts/policy-approver/templates/webhook.yaml

```diff
@@ -49,3 +49,13 @@ webhooks:
         name: {{ include "cert-manager-policy-approver.name" . }}
         namespace: {{ .Release.Namespace | quote }}
         path: /validate
+---
+apiVersion: v1
+kind: Secret
+metadata:
```

Comment:
I guess we'll also want to make the namespace configurable and the same for all these resources.